<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1 квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 квартал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4 квартал</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="show_data">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>451</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Тема</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Образцы</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Плёнки</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Отчёты</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="update_bot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_xl_bot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сформировать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
